### PR TITLE
Improved build instructions

### PR DESCRIPTION
**Pull request by ordtrogen**
you need to cd into the folder before you can build

---
**By submitting this pull-request, I confirm the following:**

- I have read and understood the developer guide in [kovri-docs](https://github.com/monero-project/kovri-docs).
- I have checked that another pull-request for this purpose does not exist.
- I have considered and confirmed that this submission will be valuable to others.
- I accept that this submission may not be used and that this pull-request may be closed by the will of the maintainer.
- I give this submission freely under the BSD 3-clause license.
---

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,7 +83,8 @@ Optional:
 **Do *not* use the zip file from github: do a recursive clone only**
 
 ```bash
-$ git clone --recursive https://github.com/monero-project/kovri
+$ git clone --recursive https://github.com/monero-project/kovri.git
```
Comment (anonimal) [inline README.md]: >kovri.git

Thought technically ok, it's not necessary with github.

File: README.md [hunk 2]
```diff
@@ -83,7 +83,8 @@ Optional:
...
+$ cd kovri
```
Comment (anonimal) [inline README.md]: Can you please change this to:

```bash
$ cd kovri && make  # see the Makefile for all build options
```

File: README.md (hunk 1 above)
Comment (ordtrogen) [reply README.md]: Sure, I know it's redundant but I'm so used to seeing the extension written out that I added it without thinking too much. Looking through the monero-project repos, I see it written out in monero-gui but not in monero. The URL you get from the GitHub "Clone or download button" does contain the .git extension.

As I'll try to modify this PR anyway, you think it's better w/ or w/o the .git extension?

Comment (anonimal) [reply README.md]: Preferably without but it's not a huge issue.

**Comment by anonimal** [comment]
Can you `$ git rebase -i` and `fixup` the commits into the original commit?

**Pull request closed without merging**